src/main.c

```diff
@@ -91,6 +92,13 @@ int main(int argc, char** argv)
 		return -1;
 	}
 
+    registerReturn = UT_register_APIDEF_l3_tests();
```

Comment:
Alignment issue. Please check the whole file.

Reply:
Alignment issue corrected, and double checked.